quickwit/quickwit-indexing/src/split_store/split_store_quota.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2022 Quickwit, Inc.
+//
+// Quickwit is offered under the AGPL v3.0 and as commercial software.
+// For commercial licensing, contact us at hello@quickwit.io.
+//
+// AGPL:
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+use byte_unit::Byte;
+use quickwit_config::IndexerConfig;
+use tracing::warn;
+
+/// A struct for keeping in check multiple SplitStore.
+#[derive(Debug)]
+pub struct SplitStoreQuota {
+    /// Current number of splits in the cache.
+    num_splits_in_cache: usize,
```

Comment:
I was reading some code in tokio and they consistently use the word `capacity` to track use:
```
    num_splits_capacity: usize,
    num_bytes_capacity: Byte,
    num_splits_capacity: usize,
    max_num_bytes_capacity: Byte
```

Reply:
That's very poor naming in my opinion.
Capacity is a very good name that has a different meaning: how much stuff you can store without extra effort.

That's how it is used in real life and in the standard library.